koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -909,7 +909,16 @@ def populate_pvc_table(self, cluster, pvcs):
         """Get or create an entry in the OCP cluster table."""
         LOG.info(log_json(msg="populating reporting_ocp_pvcs table", schema=self.schema, cluster=cluster))
         for pvc in pvcs:
-            OCPPVC.objects.get_or_create(persistent_volume=pvc[0], persistent_volume_claim=pvc[1], cluster=cluster)
+            try:
+                ocppvc = OCPPVC.objects.get(persistent_volume=pvc[0], persistent_volume_claim=pvc[1], cluster=cluster)
+                # Update the existing record's csi_volume_handle
+                ocppvc.csi_volume_handle = pvc[2]
+                ocppvc.save()
```

Comment:
I think this is probably a fine solution, but it would be good to only save this row if we needed to change it:

```suggestion
                if not ocppvc.csi_volume_handle:
                    # Update the existing record's csi_volume_handle
                    ocppvc.csi_volume_handle = pvc[2]
                    ocppvc.save(update_fields=["csi_volume_handle"])
```

🤔 we only need to set field this because it might be null right?

Reply:
Correct, when we run the migration it will be null for existing rows, and will remain null until the customer updates their operator. This allows us to set the csi volume handle whenever the information exists.